database/migrations/2015_01_01_130000_create_zips_table.php

```diff
@@ -16,8 +16,8 @@ public function up()
             $table->foreignIdFor(State::class);
             $table->foreignIdFor(Region::class)->nullable();
             $table->foreignIdFor(Timezone::class)->nullable();
-            $table->decimal('latitude', 4, 2)->nullable();
-            $table->decimal('longitude', 5, 2)->nullable();
+            $table->decimal('latitude', 9, 6)->nullable();
+            $table->decimal('longitude', 8, 6)->nullable();
```

Comment:
I don't know if a zip code can have that specific of a longitude & latitude but I will see what I can get in the database